roles/openshift_health_checker/openshift_checks/etcd_volume.py

```diff
@@ -0,0 +1,57 @@
+"""A health check for OpenShift clusters."""
+
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException, get_var
+
+
+class EtcdVolume(OpenShiftCheck):
+    """Ensures etcd storage usage does not exceed a given threshold."""
+
+    name = "etcd_volume"
+    tags = ["etcd", "health"]
+
+    # Default device usage threshold. Value should be in the range [0, 100].
+    default_threshold_percent = 90
+    # Where to find ectd data, higher priority first.
+    supported_mount_paths = ["/var/lib/etcd", "/var/lib", "/var", "/"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        # TODO: only execute this check on hosts in the 'ectd' group?
+        # Maybe also 'masters' if there are no standalone etcd hosts?
+        return super(EtcdVolume, cls).is_active(task_vars)
+
+    def run(self, tmp, task_vars):
+        mount_info = self._etcd_mount_info(task_vars)
+        available = mount_info["size_available"]
+        total = mount_info["size_total"]
+        used = total - available
+
+        threshold = get_var(
+            task_vars,
+            "r_openshift_health_checker_etcd_device_usage_threshold_percent",
```

Comment:
Example inventories are e.g. https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.ose.example - yes, I think we should start adding lines to these explaining check parameters.

I think it would be cleanest to do the conversion from external variables to internal in a vars file in the role - gives you one place to look to see what's coming in. I think we can skip translating globally available ansible stuff like `groups` and `ansible_ssh_host` and that sort of thing, but should still translate `openshift_deployment_type` and stuff introduced to `openshift.common.*` by `openshift_facts`.

That does mean in the long run we would scour all the checks for what they're using and explicitly name them in that one vars file. I think that would be a separate PR that we open up for discussion so the other ansible devs can see what an actual implementation would look like to confirm this is what we want.

For the purposes of this PR I'd say let's just do that for this check's inventory parameters and get this merged.